include/dosbox.h

```diff
@@ -74,6 +74,14 @@ void DOSBOX_Init(void);
 
 void DOSBOX_SetMachineTypeFromConfig(Section_prop* section);
 
+int64_t DOSBOX_GetTicksDone();
+void DOSBOX_SetTicksDone(const int64_t ticks_done);
+void DOSBOX_SetTicksScheduled(const int64_t ticks_scheduled);
+
+class Config;
+using ConfigPtr = std::unique_ptr<Config>;
+extern ConfigPtr control;
```

Comment:
I moved this to control.h in https://github.com/dosbox-staging/dosbox-staging/commit/a53addf59fcbd4013a1d52169356d545cb068132 to avoid the forward declaration of the Config class.

Recommend removing this and  doing`#include "control.h"` wherever you need this.

Reply:
Good tip, done.